programs/test/benchmark.c

```diff
@@ -732,6 +736,42 @@ int main(int argc, char *argv[])
         mbedtls_aes_free(&aes);
     }
 #endif
+#if defined(MBEDTLS_CIPHER_MODE_CFB)
+    if (todo.aes_cfb128) {
+        int keysize;
+        size_t iv_off = 0;
+        mbedtls_aes_context aes;
```

Comment:
Our coding style would put a blank line between variable declarations and the body of the block after this line. That's not consistently done in this file, but where it is done the readability is definitely improved, so although it's minor can we add a line before `mbedtls_aes_init(&aes)`